torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1675,6 +1676,15 @@ def __init__(
         # symbols that are marked dynamic
         self._marked_dynamic = marked_dynamic
 
+        # track unsupported sympy functions
+        self._supported_sympy_functions: Set[sympy.Function] = {
+            Mod,
+            PythonMod,
+            FloorDiv,
+        }
+        self._unsupported_sympy_functions: Set[sympy.Function] = {}
```

Comment:
Adjust the initializations and comments here, `_supported_sympy_functions` is a fixed list and `_unsupported_sympy_functions` will be immediately populated by enumeration? So maybe no need to initialize `_unsupported_sympy_functions` here or just inline the enumeration.